### ISSUE TITLE
load/save round trip fails

### ISSUE THREAD

**Issue by evmar** [automerge/automerge#63]
```js
let doc = automerge.init();
doc = automerge.change(doc, 'init', (doc) => {
  doc.foo = [];
});
doc = automerge.load(automerge.save(doc));
doc = automerge.change(doc, 'add', (doc) => {
  doc.foo.push(1);
});
doc = automerge.load(automerge.save(doc));
```

Fails with `Error: Missing index entry for list element undefined:1`

The docs are kinda unclear on how `save()` is supposed to work -- it says that save returns a string, but then the first example wraps it in a JSON.stringify.  In my experiments it looks like save maybe just returns the thing you handed it?

**Comment by ept** [comment]
Hi @evmar, sorry for the delay. The code example you give should work, and indeed I added it as a test case and it works: da79989eb75a3fdc09ea5575b51f68c6e04e27be — are you using the latest version of Automerge?

JSON.stringify is not needed with Automerge.save(). The example in the README that used JSON.stringify was supposed to demonstrate what you would do *without* Automerge. However, I see how that would be confusing, so I've just removed that part of the example (it's not really important anyway).
